<!DOCTYPE html>
<html lang="en-us">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.4.0">

  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Drew Steen">

  
  
  
    
  
  <meta name="description" content="SteenLab website">

  
  <link rel="alternate" hreflang="en-us" href="https://adsteen.github.io/">

  


  

  
  
  
  <meta name="theme-color" content="#3f51b5">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.0/css/all.css" integrity="sha384-aOkxzJ5uQz7WBObEZcHvV5JvRW3TUc2rNPA7pe3AwnsUohiw1Vj2Rgx2KSOkF5+h" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.css" integrity="sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ==" crossorigin="anonymous">
    

    

  

  
  
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:400,400italic,700|Roboto+Mono&display=swap">
  

  
  
  
  <link rel="stylesheet" href="/css/academic.min.ce53f5d1ff75f9f89bb1c243b514bb45.css">

  

  
  
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-60056958-1', 'auto');
      
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="https://www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  
  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="Steen Lab">
  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon-32.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="https://adsteen.github.io/">

  
  
  
  
    
  
  
  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@drdrewsteen">
  <meta property="twitter:creator" content="@drdrewsteen">
  
  <meta property="og:site_name" content="Steen Lab">
  <meta property="og:url" content="https://adsteen.github.io/">
  <meta property="og:title" content="Steen Lab">
  <meta property="og:description" content="SteenLab website"><meta property="og:image" content="https://adsteen.github.io/img/lab_logo.png">
  <meta property="twitter:image" content="https://adsteen.github.io/img/lab_logo.png"><meta property="og:locale" content="en-us">
  
    <meta property="og:updated_time" content="2030-06-01T13:00:00&#43;00:00">
  

  


  





  <title>Steen Lab</title>

</head>

<body id="top" data-spy="scroll" data-offset="70" data-target="#navbar-main" >

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  
<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/">Steen Lab</a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav mr-auto">
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#projects" data-target="#projects"><span>Projects</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#people" data-target="#people"><span>People</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#featured" data-target="#featured"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#contact" data-target="#contact"><span>Contact</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/files/cv.pdf"><span>CV</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        

        <li class="nav-item">
          <a class="nav-link " href="/lab-resources"><span>Lab resources</span></a>
        </li>

        
        

      
      </ul>
      <ul class="navbar-nav ml-auto">
      

        

        
        <li class="nav-item">
          <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        

        
        <li class="nav-item">
          <a class="nav-link js-dark-toggle" href="#"><i class="fas fa-moon" aria-hidden="true"></i></a>
        </li>
        

      </ul>

    </div>
  </div>
</nav>


  


<span class="js-widget-page d-none"></span>




  







  
  
  

  

  

  
    
    
      
    
    
    
  

  
    
    
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="demo" class="home-section wg-blank dark parallax " style="background-image: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), url(&#39;https://adsteen.github.io/img/WOR_pano.jpg&#39;);padding: 20px 0 20px 0;" >
    <div class="container">
      


<div class="row">
  
    <div class="col-lg-12">
      <h1>Welcome to the Steen Lab</h1>
      
      <p>We study the biology of heterotrophic microbes in aquatic environments in order to understand their effect on the Earth&rsquo;s geochemical cycles.</p>
<!-- raw HTML omitted -->
<!-- raw HTML omitted -->

    </div>
  
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="projects" class="home-section wg-portfolio   "  >
    <div class="container">
      










<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">

    <h1>Projects</h1>
    

  </div>
  <div class="col-xs-12 col-md-8">



    

    

    <div class=" row js-layout-row project-showcase mt-5">
      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item js-id-bioinformatics" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0" itemprop="name"><a href="/project/meta-analyses/"  itemprop="url">Bioinformatic meta-analyses</a></h3>

      
      <div class="article-style" itemprop="articleBody">
        Meta-analyses of public sequence data to better understand the structure of microbial life
      </div>
      

      <div class="btn-links">
        








  



















      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
      
      <a href="/project/meta-analyses/" >
        <img src="/project/meta-analyses/featured_huf6dbf77abf96a8c35d8138b0154cd975_499376_540x0_resize_lanczos_2.png" itemprop="image" alt="">
      </a>
      
    </div>
  </div>
</div>

        

      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item js-id-geochemistry" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0" itemprop="name"><a href="/project/extracellular_enzymes/"  itemprop="url">Extracellular enzymes</a></h3>

      
      <div class="article-style" itemprop="articleBody">
        Geochemistry and microbial ecology of extracellular enzymes in aquatic environemnts
      </div>
      

      <div class="btn-links">
        








  



















      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
      
      <a href="/project/extracellular_enzymes/" >
        <img src="/project/extracellular_enzymes/featured_hu66e42e98c9aeb8597cf57cf9b7004a3f_436864_540x0_resize_q90_lanczos.jpg" itemprop="image" alt="">
      </a>
      
    </div>
  </div>
</div>

        

      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item js-id-outreach" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0" itemprop="name"><a href="/project/outreach/"  itemprop="url">Outreach</a></h3>

      
      <div class="article-style" itemprop="articleBody">
        Involving high school students in authentic geomicrobiology research
      </div>
      

      <div class="btn-links">
        








  



















      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
      
      <a href="/project/outreach/" >
        <img src="/project/outreach/featured_hu64a3023eb095663e8120138ecdddf6e2_1415945_540x0_resize_q90_lanczos.jpg" itemprop="image" alt="">
      </a>
      
    </div>
  </div>
</div>

        

      

        
        
        

        
          




  








  







<div class="col-lg-12 project-item isotope-item js-id-geochemistry" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row align-items-center">
    <div class="col-12 col-md-6">
      <h3 class="article-title mb-0 mt-0" itemprop="name"><a href="/project/permafrost/"  itemprop="url">Thawing permafrost</a></h3>

      
      <div class="article-style" itemprop="articleBody">
        Geochemistry and microbial ecology of thawing permafrost in Svalbard
      </div>
      

      <div class="btn-links">
        








  



















      </div>

    </div>
    <div class="col-12 col-md-6 order-first ">
      
      
      
      <a href="/project/permafrost/" >
        <img src="/project/permafrost/featured_huf97279350ae28e09faa0a6944c7771ac_2239969_540x0_resize_q90_lanczos.JPG" itemprop="image" alt="">
      </a>
      
    </div>
  </div>
</div>

        

      
    </div>

  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="posts" class="home-section wg-pages   "  >
    <div class="container">
      








  























  





  


<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Recent Posts</h1>
    
  </div>
  <div class="col-12 col-lg-8">

    

    
      
        










  





  


<div class="media stream-item" itemscope itemtype="http://schema.org/BlogPosting">
  <div class="media-body">

    <h3 class="article-title mb-0 mt-0" itemprop="name">
      <a href="/post/on-the-fallacy-of-the-work-life-balance/"  itemprop="url">On the fallacy of the work-life balance</a>
    </h3>

    
    <div class="article-style" itemprop="articleBody">
      A little while ago I was asked by C-DEBI to give a webinar* for their Professional Development Series on work-life balance. Now, I …
    </div>
    

    <div class="stream-meta article-metadata">

      

      
        



<meta content="2019-11-22 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-11-22 09:43:51 -0500 EST" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
          Last updated on
      
    
    <time>Nov 22, 2019</time>
  </span>
  

  

  

  
  
  

  
  

  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

      
    

    

  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="people" class="home-section wg-people   "  >
    <div class="container">
      


<div class="row justify-content-center people-widget">
  
  <div class="col-md-12 section-heading">
    <h1>People</h1>
    
  </div>
  

  

  
  <div class="col-md-12">
    <h2 class="mb-4">Principal Investigator</h2>
  </div>

  
  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/admin/"><img class="portrait" src="/authors/admin/avatar_huf9143222a0e4b4beb788d1b7752ba92e_118478_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/admin/">Drew Steen</a></h2>
      <h3>Assistant Professor of Microbiology and Earth and Planetary Sciences</h3>
      <p class="people-interests">Geomicrobiology, Marine organic geochemistry, Bioinformatics, Machine learning</p>
    </div>
  </div>

  
  
  <div class="col-md-12">
    <h2 class="mb-4">Grad Students</h2>
  </div>

  
  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/bgreen/"><img class="portrait" src="/authors/bgreen/avatar_hudd5e2792e827b66d5b392282f962ab11_1159630_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/bgreen/">Brianna Green</a></h2>
      <h3>Master&rsquo;s student</h3>
      
    </div>
  </div>

  
  
  <div class="col-md-12">
    <h2 class="mb-4">Undergrads</h2>
  </div>

  
  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/alonergan/"><img class="portrait" src="/authors/alonergan/avatar_hu9839c73185d1be2544cfedaeb4fb3fc1_265680_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/alonergan/">Andrew Lonergan</a></h2>
      <h3>undergrad</h3>
      <p class="people-interests">Geomicrobiology, Extracellular enzymes</p>
    </div>
  </div>

  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/mmiles/"><img class="portrait" src="/authors/mmiles/avatar_hu0943c85463a0931c34c89c4d67e34eda_445957_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/mmiles/">Maryn Miles</a></h2>
      <h3>undergrad</h3>
      <p class="people-interests">Geomicrobiology, Extracellular enzymes</p>
    </div>
  </div>

  
  
  <div class="col-md-12">
    <h2 class="mb-4">Alumni</h2>
  </div>

  
  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/abarrett/"><img class="portrait" src="/authors/abarrett/avatar_hu52a603635ecebd45650b162dadabb4e5_12861_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/abarrett/">Abigail Barrett</a></h2>
      <h3>Masters&rsquo; student</h3>
      <p class="people-interests">Environmental consulting, Data science</p>
    </div>
  </div>

  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/ccook62/"><img class="portrait" src="/authors/ccook62/avatar_hu4e982002363bdcdffed8907231081afb_49357_150x150_fill_q90_lanczos_center.JPG" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/ccook62/">Chris Cook</a></h2>
      <h3>Masters&rsquo; student</h3>
      <p class="people-interests">Geomicrobiology, Harmful algal blooms, Microcystin</p>
    </div>
  </div>

  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/lmullen/"><img class="portrait" src="/authors/lmullen/avatar_hu415ba60bdea298a9735776c66da1b147_3564095_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/lmullen/">Lauren Mullen</a></h2>
      <h3>Masters&rsquo; student</h3>
      <p class="people-interests">Microbial Ecology, Extracellular enzymes, Health microbiology</p>
    </div>
  </div>

  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/lquigley/"><img class="portrait" src="/authors/lquigley/avatar_hu7576341246c0261596e558b7a7cc2be6_174967_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/lquigley/">Lauren N. M. Quigley</a></h2>
      <h3>Ph.D. student</h3>
      <p class="people-interests">Aromatic carbon metabolism, Metagenomics, Metatranscriptomics</p>
    </div>
  </div>

  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/pmather/"><img class="portrait" src="/authors/pmather/avatar_huf4fcc30fd3ad214737f1249198aaab60_1151107_150x150_fill_q90_lanczos_center.jpeg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/pmather/">P. Drew Mather</a></h2>
      <h3>Undergraduate student</h3>
      <p class="people-interests">Natural resource management, Water availability law and policy</p>
    </div>
  </div>

  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/rpinals/"><img class="portrait" src="/authors/rpinals/avatar_hu386d518dc93394a45f4a8b8857e0b220_1104293_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/rpinals/">Rebecca L. Pinals</a></h2>
      <h3>Visiting student</h3>
      <p class="people-interests">Nanomaterials in biological environments, Nanosensor development</p>
    </div>
  </div>

  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/rspaulding/"><img class="portrait" src="/authors/rspaulding/avatar_hu989ac161d89bf4c0f16f54d06aab80a3_239914_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/rspaulding/">Russell Spaulding II</a></h2>
      <h3>undergrad</h3>
      <p class="people-interests">STEM teaching, Microplastics</p>
    </div>
  </div>

  

  
  
  
  
    
  
  <div class="col-12 col-sm-auto people-person">
    
    
      
      
    
    
      <a href="/authors/zstooksbury/"><img class="portrait" src="/authors/zstooksbury/avatar_hu47ae0d2743b8fa339caf7634c358d8a6_929230_150x150_fill_q90_lanczos_center.jpg" alt="Avatar"></a>
    

    <div class="portrait-title">
      <h2><a href="/authors/zstooksbury/">Zachery Stooksbury</a></h2>
      <h3>undergrad</h3>
      <p class="people-interests">Biogeochemistry, Aqueous Geochemistry, GIS, Marine Sciences</p>
    </div>
  </div>

  
  
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="featured" class="home-section wg-featured   "  >
    <div class="container">
      























<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Featured Publications</h1>
    
  </div>
  <div class="col-12 col-lg-8">

    

    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-09-02 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2020-09-07 10:36:57 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Marine Science
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2020_steen_hwk_review/">
      <img src="/publication/2020_steen_hwk_review/featured_hu8a24f0b379a83bcd6ecf6a333cfe3935_540432_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2020_steen_hwk_review/" itemprop="url">Analytical and Computational Advances, Opportunities, and Challenges in Marine Organic Biogeochemistry in an Era of “Omics”</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Advances in sampling tools, analytical methods, and data handling capabilities have been fundamental to the growth of marine organic biogeochemistry over the past four decades. There has always been a strong feedback between analytical advances and scientific advances. However, whereas advances in analytical technology were often the driving force that made possible progress in elucidating the sources and fate of organic matter in the ocean in the first decades of marine organic biogeochemistry, today process-based scientific questions should drive analytical developments. Several paradigm shifts and challenges for the future are related to the intersection between analytical progress and scientific evolution. Untargeted “molecular headhunting” for its own sake is now being subsumed into process-driven targeted investigations that ask new questions and thus require new analytical capabilities. However, there are still major gaps in characterizing the chemical composition and biochemical behavior of macromolecules, as well as in generating reference standards for relevant types of organic matter. Field-based measurements are now routinely complemented by controlled laboratory experiments and in situ rate measurements of key biogeochemical processes. And finally, the multidisciplinary investigations that are becoming more common generate large and diverse datasets, requiring innovative computational tools to integrate often disparate data sets, including better global coverage and mapping. Here, we compile examples of developments in analytical methods that have enabled transformative scientific advances since 2004, and we project some challenges and opportunities in the near future. We believe that addressing these challenges and capitalizing on these opportunities will ensure continued progress in understanding the cycling of organic carbon in the ocean.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2020_Steen_HWK_review.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020_steen_hwk_review/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.frontiersin.org/articles/10.3389/fmars.2020.00718/full" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fmars.2020.00718" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-02-01 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2020-06-06 12:48:10 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>February 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Deep-Sea Research I
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2020_mahmoudi_med/">
      <img src="/publication/2020_mahmoudi_med/featured_hu8f5e8ece1545cdc7772ac0a2783c4f41_98046_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2020_mahmoudi_med/" itemprop="url">Patterns in extracellular enzyme activity and microbial diversity in deep-sea Mediterranean sediments</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Deep-sea sediments are populated by diverse microbial communities that derive their nutritional requirements from the degradation of organic matter. Extracellular hydrolytic enzymes play a key role in the survival of microbes by enabling them to access and degrade complex organic compounds that are found in seafloor sediments. Despite their importance, extracellular enzymatic activity is poorly characterized at water depths greater than a few hundred meters where physical properties, such as pressure and temperature, create a unique environment for influencing enzyme behavior. Here, we investigated microbial communities and enzyme activities in surface sediment collected at four sampling stations in the central Mediterranean Sea at water depths ranging from 800 to 2200 m. Fluorometric assays revealed that extracellular hydrolytic activity varied according to substrate type and water depth which suggests that the distributions of these enzymes within this basin are not homogenous. Furthermore, enzyme activities indicated substantial demand for phosphomonoesters and proteins, with measurable but much lower demand for polysaccharides. Barcoded amplicon sequencing of bacterial and archaeal SSU genes revealed that microbial communities varied across sampling stations and some groups displayed water-depth related trends. Our results demonstrate that heterotrophic capabilities of microbes in deep-sea Mediterranean sediments can differ substantially even within the same region.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2020_Mahmoudi_Med.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020_mahmoudi_med/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www-sciencedirect-com.proxy.lib.utk.edu/science/article/pii/S0967063720300194" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.dsr.2020.103231" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-09-17 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-17 14:45:39 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/troyalty/">Taylor Royalty</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      arXiv
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_senter_alignment/" itemprop="url">Unaligned Sequence Similarity Search Using Deep Learning</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Gene annotation has traditionally required direct comparison of DNA sequences between an unknown gene and a database of known ones using string comparison methods. However, these methods do not provide useful information when a gene does not have a close match in the database. In addition, each comparison can be costly when the database is large since it requires alignments and a series of string comparisons. In this work we propose a novel approach: using recurrent neural networks to embed DNA or amino-acid sequences in a low-dimensional space in which distances correlate with functional similarity. This embedding space overcomes both shortcomings of the method of aligning sequences and comparing homology. First, it allows us to obtain information about genes which do not have exact matches by measuring their similarity to other ones in the database. If our database is labeled this can provide labels for a query gene as is done in traditional methods. However, even if the database is unlabeled it allows us to find clusters and infer some characteristics of the gene population. In addition, each comparison is much faster than traditional methods since the distance metric is reduced to the Euclidean distance, and thus efficient approximate nearest neighbor algorithms can be used to find the best match. We present results showing the advantage of our algorithm. More specifically we show how our embedding can be useful for both classification tasks when our labels are known, and clustering tasks where our sequences belong to classes which have not been seen before.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Senter.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_senter_alignment/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://arxiv.org/abs/1909.06929" target="_blank" rel="noopener">
  Source Document
</a>




  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-08-06 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-09 21:40:53 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>August 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      ISME Journal
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2019_steen_et_al_isme/">
      <img src="/publication/2019_steen_et_al_isme/featured_hufb273d8ecccadd1f77b053cfe1574efb_1751387_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_steen_et_al_isme/" itemprop="url">High proportions of bacteria and archaea across most biomes remain uncultured</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    A recent paper by Martiny argues that “high proportions” of bacteria in diverse Earth environments have been cultured. Here we reanalyze a portion of the data in that paper, and argue that the conclusion is based on several technical errors, most notably a calculation of sequence similarity that does not account for sequence gaps, and the reliance on 16S rRNA gene amplicons that are known to be biased towards cultured organisms. We further argue that the paper is also based on a conceptual error: namely, that sequence similarity cannot be used to infer “culturability” because one cannot infer physiology from 16S rRNA gene sequences. Combined with other recent, more reliable studies, the evidence supports the conclusion that most bacterial and archaeal taxa remain uncultured.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Steen_et_al_Uncultured.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_steen_et_al_isme/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41396-019-0484-y" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41396-019-0484-y" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-08-06 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-26 10:39:53 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/troyalty/">Taylor Royalty</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>August 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Applied and Environmental Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2019_steen_et_al_wor_enz/">
      <img src="/publication/2019_steen_et_al_wor_enz/featured_hu1656548ef733a7c2dffbc57534bef85b_1526248_918x517_fill_q90_lanczos_smart1.jpeg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_steen_et_al_wor_enz/" itemprop="url">Kinetics and Identities of Extracellular Peptidases in Subsurface Sediments of the White Oak River Estuary, North Carolina</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Anoxic subsurface sediments contain communities of heterotrophic microorganisms that metabolize organic carbon at extraordinarily slow rates. In order to assess the mechanisms by which subsurface microorganisms access detrital sedimentary organic matter, we measured kinetics of a range of extracellular peptidases in anoxic sediments of the White Oak River estuary, NC. Nine distinct peptidase substrates were enzymatically hydrolyzed at all depths. Potential peptidase activities (Vmax) decreased with increasing sediment depth, although Vmax expressed on a per cell basis was approximately the same at all depths. Half-saturation constants (Km) decreased with depth, indicating peptidases that functioned more efficiently at low substrate concentrations. Potential activities of extracellular peptidases acting on molecules that are enriched in degraded organic matter (D-phenylalanine and L-ornithine) increased relative to enzymes that act on L-phenylalanine, further suggesting microbial community adaptation to access degraded organic matter. Nineteen classes of predicted, exported peptidases were identified in genomic data from the same site, of which genes for class C25 (gingipain-like) peptidases represented more than 40% at each depth. Methionine aminopeptidases, zinc carboxypeptidases, and class S24-like peptidases, which are involved in single-stranded DNA repair, were also abundant. These results suggest a subsurface heterotrophic microbial community that primarily accesses low-quality detrital organic matter via a diverse suite of well-adapted extracellular enzymes.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Steen_et_al_WOR.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_steen_et_al_wor_enz/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/adsteen/WOR_enz_2013_2014" target="_blank" rel="noopener">
  Code
</a>












<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41396-019-0484-y" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-03-06 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-06 09:40:34 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/lmullen/">Lauren Mullen</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>March 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2018_mullen_et_al_mxshs/">
      <img src="/publication/2018_mullen_et_al_mxshs/featured_hu98982b0bb24fd2372dbd1414c5739cb7_1523384_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2018_mullen_et_al_mxshs/" itemprop="url">Potential Activities of Freshwater Exo- and Endo-Acting Extracellular Peptidases in East Tennessee and the Pocono Mountains</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Proteins constitute a particularly bioavailable subset of organic carbon and nitrogen in aquatic environments but must be hydrolyzed by extracellular enzymes prior to being metabolized by microorganisms. Activities of extracellular peptidases (protein-degrading enzymes) have frequently been assayed in freshwater systems, but such studies have been limited to substrates for a single enzyme [leucyl aminopeptidase (Leu-AP)] out of more than 300 biochemically recognized peptidases. Here, we report kinetic measurements of extracellular hydrolysis of five substrates in 28 freshwater bodies in the Delaware Water Gap National Recreation Area in the Pocono Mountains (PA, United States) and near Knoxville (TN, United States), between 2013 and 2016. The assays putatively test for four aminopeptidases (arginyl aminopeptidase, glyclyl aminopeptidase, Leu-AP, and pyroglutamyl aminopeptidase), which cleave N-terminal amino acids from proteins, and trypsin, an endopeptidase, which cleaves proteins mid-chain. Aminopeptidase and the trypsin-like activity were observed in all water bodies, indicating that a diverse set of peptidases is typical in freshwater. However, ratios of peptidase activities were variable among sites: aminopeptidases dominated at some sites and trypsin-like activity at others. At a given site, the ratios remained fairly consistent over time, indicating that they are driven by ecological factors. Studies in which only Leu-AP activity is measured may underestimate the total peptidolytic capacity of an environment, due to the variable contribution of endopeptidases.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2018_Mullen_et_al_PEEC_TN.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2018_mullen_et_al_mxshs/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fmicb.2018.00368" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2015-06-10 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-04 17:29:16 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2015</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      FASEB Journal
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2015_michalska_et_al/">
      <img src="/publication/2015_michalska_et_al/featured_huca046fa4896c492d48fe98b6e5dfaf4f_220553_918x517_fill_q90_lanczos_smart1.jpeg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2015_michalska_et_al/" itemprop="url">New aminopeptidase from “microbial dark matter” archaeon</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Marine sediments host a large population of diverse, heterotrophic, uncultured microorganisms with unknown physiologies that control carbon flow through organic matter decomposition. Recently, single-cell genomics uncovered new key players in these processes, such as the miscellaneous crenarchaeotal group. These widespread archaea encode putative intra- and extracellular proteases for the degradation of detrital proteins present in sediments. Here, we show that one of these enzymes is a self-compartmentalizing tetrameric aminopeptidase with a preference for cysteine and hydrophobic residues at the N terminus of the hydrolyzed peptide. The ability to perform detailed characterizations of enzymes from native subsurface microorganisms, without requiring that those organisms first be grown in pure culture, holds great promise for understanding key carbon transformations in the environment as well as identifying new enzymes for biomedical and biotechnological applications.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2015_Michalska_et_al_Bathyaminopeptidase.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2015_michalska_et_al/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1096/fj.15-272906" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="" itemprop="datePublished">
<meta content="2019-09-06 09:40:34 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 0001</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2018_lloyd_et_al_uncultured/">
      <img src="/publication/2018_lloyd_et_al_uncultured/featured_hu483438cd82e3753239ac177b1c5d1640_114970_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2018_lloyd_et_al_uncultured/" itemprop="url">Phylogenetically Novel Uncultured Microbial Cells Dominate Earth Microbiomes</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    To describe a microbe’s physiology, including its metabolism, environmental roles, and growth characteristics, it must be grown in a laboratory culture. Unfortunately, many phylogenetically novel groups have never been cultured, so their physiologies have only been inferred from genomics and environmental characteristics. Although the diversity, or number of different taxonomic groups, of uncultured clades has been studied well, their global abundances, or numbers of cells in any given environment, have not been assessed. We quantified the degree of similarity of 16S rRNA gene sequences from diverse environments in publicly available metagenome and metatranscriptome databases, which we show have far less of the culture bias present in primer-amplified 16S rRNA gene surveys, to those of their nearest cultured relatives. Whether normalized to scaffold read depths or not, the highest abundances of metagenomic 16S rRNA gene sequences belong to phylogenetically novel uncultured groups in seawater, freshwater, terrestrial subsurface, soil, hypersaline environments, marine sediment, hot springs, hydrothermal vents, nonhuman hosts, snow, and bioreactors (22% to 87% uncultured genera to classes and 0% to 64% uncultured phyla). The exceptions were human and human-associated environments, which were dominated by cultured genera (45% to 97%). We estimate that uncultured genera and phyla could comprise 7.3 × 10<!-- raw HTML omitted -->29<!-- raw HTML omitted --> (81%) and 2.2 × 10<!-- raw HTML omitted -->29<!-- raw HTML omitted --> (25%) of microbial cells, respectively. Uncultured phyla were overrepresented in metatranscriptomes relative to metagenomes (46% to 84% of sequences in a given environment), suggesting that they are viable. Therefore, uncultured microbes, often from deeply phylogenetically divergent groups, dominate nonhuman environments on Earth, and their undiscovered physiologies may matter for Earth systems.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2018_Lloyd_Steen_et_al_Uncultured_Majority.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2018_lloyd_et_al_uncultured/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1128/mSystems.00055-18" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="" itemprop="datePublished">
<meta content="2020-09-03 21:38:35 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 0001</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2020_murray_roadmap/" itemprop="url">Roadmap for naming uncultivated Archaea and Bacteria</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The assembly of single-amplified genomes (SAGs) and metagenome-assembled genomes (MAGs) has led to a surge in genome-based discoveries of members affiliated with Archaea and Bacteria, bringing with it a need to develop guidelines for nomenclature of uncultivated microorganisms. The International Code of Nomenclature of Prokaryotes (ICNP) only recognizes cultures as ‘type material’, thereby preventing the naming of uncultivated organisms. In this Consensus Statement, we propose two potential paths to solve this nomenclatural conundrum. One option is the adoption of previously proposed modifications to the ICNP to recognize DNA sequences as acceptable type material; the other option creates a nomenclatural code for uncultivated Archaea and Bacteria that could eventually be merged with the ICNP in the future. Regardless of the path taken, we believe that action is needed now within the scientific community to develop consistent rules for nomenclature of uncultivated taxa in order to provide clarity and stability, and to effectively communicate microbial diversity.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2020_Murray_roadmap.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020_murray_roadmap/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41564-020-0733-x" target="_blank" rel="noopener">
  Source Document
</a>




  </div>
  

</div>

      
    

  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="publications" class="home-section wg-pages   "  >
    <div class="container">
      








  























  





  


<div class="row">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Lab Publications</h1>
    
  </div>
  <div class="col-12 col-lg-8">

    <!-- raw HTML omitted -->


    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-09-02 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2020-09-07 10:36:57 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Marine Science
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2020_steen_hwk_review/">
      <img src="/publication/2020_steen_hwk_review/featured_hu8a24f0b379a83bcd6ecf6a333cfe3935_540432_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2020_steen_hwk_review/" itemprop="url">Analytical and Computational Advances, Opportunities, and Challenges in Marine Organic Biogeochemistry in an Era of “Omics”</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Advances in sampling tools, analytical methods, and data handling capabilities have been fundamental to the growth of marine organic biogeochemistry over the past four decades. There has always been a strong feedback between analytical advances and scientific advances. However, whereas advances in analytical technology were often the driving force that made possible progress in elucidating the sources and fate of organic matter in the ocean in the first decades of marine organic biogeochemistry, today process-based scientific questions should drive analytical developments. Several paradigm shifts and challenges for the future are related to the intersection between analytical progress and scientific evolution. Untargeted “molecular headhunting” for its own sake is now being subsumed into process-driven targeted investigations that ask new questions and thus require new analytical capabilities. However, there are still major gaps in characterizing the chemical composition and biochemical behavior of macromolecules, as well as in generating reference standards for relevant types of organic matter. Field-based measurements are now routinely complemented by controlled laboratory experiments and in situ rate measurements of key biogeochemical processes. And finally, the multidisciplinary investigations that are becoming more common generate large and diverse datasets, requiring innovative computational tools to integrate often disparate data sets, including better global coverage and mapping. Here, we compile examples of developments in analytical methods that have enabled transformative scientific advances since 2004, and we project some challenges and opportunities in the near future. We believe that addressing these challenges and capitalizing on these opportunities will ensure continued progress in understanding the cycling of organic carbon in the ocean.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2020_Steen_HWK_review.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020_steen_hwk_review/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.frontiersin.org/articles/10.3389/fmars.2020.00718/full" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fmars.2020.00718" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-02-01 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2020-06-06 12:48:10 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>February 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Deep-Sea Research I
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2020_mahmoudi_med/">
      <img src="/publication/2020_mahmoudi_med/featured_hu8f5e8ece1545cdc7772ac0a2783c4f41_98046_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2020_mahmoudi_med/" itemprop="url">Patterns in extracellular enzyme activity and microbial diversity in deep-sea Mediterranean sediments</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Deep-sea sediments are populated by diverse microbial communities that derive their nutritional requirements from the degradation of organic matter. Extracellular hydrolytic enzymes play a key role in the survival of microbes by enabling them to access and degrade complex organic compounds that are found in seafloor sediments. Despite their importance, extracellular enzymatic activity is poorly characterized at water depths greater than a few hundred meters where physical properties, such as pressure and temperature, create a unique environment for influencing enzyme behavior. Here, we investigated microbial communities and enzyme activities in surface sediment collected at four sampling stations in the central Mediterranean Sea at water depths ranging from 800 to 2200 m. Fluorometric assays revealed that extracellular hydrolytic activity varied according to substrate type and water depth which suggests that the distributions of these enzymes within this basin are not homogenous. Furthermore, enzyme activities indicated substantial demand for phosphomonoesters and proteins, with measurable but much lower demand for polysaccharides. Barcoded amplicon sequencing of bacterial and archaeal SSU genes revealed that microbial communities varied across sampling stations and some groups displayed water-depth related trends. Our results demonstrate that heterotrophic capabilities of microbes in deep-sea Mediterranean sediments can differ substantially even within the same region.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2020_Mahmoudi_Med.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020_mahmoudi_med/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www-sciencedirect-com.proxy.lib.utk.edu/science/article/pii/S0967063720300194" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.dsr.2020.103231" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2020-02-01 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2020-09-03 19:04:40 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>February 2020</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Deep-Sea Research I
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2020_larowe_review/" itemprop="url">The fate of organic carbon in marine sediments - New insights from recent data and analysis</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Organic carbon in marine sediments is a critical component of the global carbon cycle, and its degradation influences a wide range of phenomena, including the magnitude of carbon sequestration over geologic timescales, the recycling of inorganic carbon and nutrients, the dissolution and precipitation of carbonates, the production of methane and the nature of the seafloor biosphere. Although much has been learned about the factors that promote and hinder rates of organic carbon degradation in natural systems, the controls on the distribution of organic carbon in modern and ancient sediments are still not fully understood. In this review, we summarize how recent findings are changing entrenched perspectives on organic matter degradation in marine sediments: a shift from a structurally-based chemical reactivity viewpoint towards an emerging acceptance of the role of the ecosystem in organic matter degradation rates. That is, organic carbon has a range of reactivities determined by not only the nature of the organic compounds, but by the biological, geochemical, and physical attributes of its environment. This shift in mindset has gradually come about due to a greater diversity of sample sites, the molecular revolution in biology, discoveries concerning the extent and limits of life, advances in quantitative modeling, investigations of ocean carbon cycling under a variety of extreme paleo-conditions (e.g. greenhouse environments, euxinic/anoxic oceans), the application of novel analytical techniques and interdisciplinary efforts. Adopting this view across scientific disciplines will enable additional progress in understanding how marine sediments influence the global carbon cycle.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2020_LaRowe_review.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020_larowe_review/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.sciencedirect.com/science/article/pii/S0012825219305720?casa_token=DnUZSQqp1aoAAAAA:yc7HNqH2J_GuWkL-1H4osjtuANZfuIC3pZnqQzLtBl-1TdTsTErQZVya2CmS3mMpNgczKkqehg" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.earscirev.2020.103146" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-10-01 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-10-15 17:06:02 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/susan-q.-lang/">Susan Q. Lang</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/magdalena-r.-osburn/">Magdalena R. Osburn</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>October 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_lang_et_al_chapter/" itemprop="url">Carbon in the deep biosphere: forms, fates, and biogeochemical cycling</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    This chapter is in <em>Deep Carbon: Past to Present</em>, Beth N. Orcutt, Isabelle Daniel, and Rajdeep Gupta, eds, to be published by Cambrdige University Press in October 2019.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Lang_et_al_DCO_chapter.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_lang_et_al_chapter/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.cambridge.org/gb/academic/subjects/earth-and-environmental-science/geochemistry-and-environmental-chemistry/deep-carbon-past-present?format=HB" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1128/mBio.02376-18" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-09-17 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-17 14:45:39 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/troyalty/">Taylor Royalty</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      arXiv
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_senter_alignment/" itemprop="url">Unaligned Sequence Similarity Search Using Deep Learning</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Gene annotation has traditionally required direct comparison of DNA sequences between an unknown gene and a database of known ones using string comparison methods. However, these methods do not provide useful information when a gene does not have a close match in the database. In addition, each comparison can be costly when the database is large since it requires alignments and a series of string comparisons. In this work we propose a novel approach: using recurrent neural networks to embed DNA or amino-acid sequences in a low-dimensional space in which distances correlate with functional similarity. This embedding space overcomes both shortcomings of the method of aligning sequences and comparing homology. First, it allows us to obtain information about genes which do not have exact matches by measuring their similarity to other ones in the database. If our database is labeled this can provide labels for a query gene as is done in traditional methods. However, even if the database is unlabeled it allows us to find clusters and infer some characteristics of the gene population. In addition, each comparison is much faster than traditional methods since the distance metric is reduced to the Euclidean distance, and thus efficient approximate nearest neighbor algorithms can be used to find the best match. We present results showing the advantage of our algorithm. More specifically we show how our embedding can be useful for both classification tasks when our labels are known, and clustering tasks where our sequences belong to classes which have not been seen before.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Senter.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_senter_alignment/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://arxiv.org/abs/1909.06929" target="_blank" rel="noopener">
  Source Document
</a>




  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-09-09 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-09 14:09:10 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/troyalty/">Taylor Royalty</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      ISME Journal
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_royalty_partitioning/" itemprop="url">Kinetics and identities of extracellular peptidases in subsurface sediments of the White Oak River Estuary, NC</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Widely used microbial taxonomies, such as the NCBI taxonomy, are based on a combination of sequence homology among conserved genes and historically accepted taxonomies, which were developed based on observable traits such as morphology and physiology. A recently proposed alternative taxonomy database, the Genome Taxonomy Database (GTDB), incorporates only sequence homology of conserved genes and attempts to partition taxonomic ranks such that each rank implies the same amount of evolutionary distance, regardless of its position on the phylogenetic tree. This provides the first opportunity to completely separate taxonomy from traits and therefore to quantify how taxonomic rank corresponds to traits across the microbial tree of life. We quantified the relative abundances of clusters of orthologous group functional categories (COG-FCs) as a proxy for traits within the lineages of 13,735 cultured and uncultured microbial lineages from a custom-curated genome database. On average, 41.4% of the variation in COG-FC relative abundance is explained by taxonomic rank, with domain, phylum, class, order, family, and genus explaining, on average, 3.2%, 14.6%, 4.1%, 9.2%, 4.8%, and 5.5% of the variance, respectively (P &lt; 0.001 for all). To our knowledge, this is the first work to quantify the variance in metabolic potential contributed by individual taxonomic ranks. A qualitative comparison between the COG-FC relative abundances and genus-level phylogenies, generated from published concatenated protein sequence alignments, further supports the idea that metabolic potential is taxonomically coherent at higher taxonomic ranks. The quantitative analyses presented here characterize the integral relationship between diversification of microbial lineages and the metabolisms which they host.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Royalty_Steen_COG_partitioning.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_royalty_partitioning/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://aem.asm.org/content/early/2019/07/15/AEM.00102-19" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1128/AEM.00102-19" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-08-07 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-09 21:35:15 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>August 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Elementa: Science of the Anthropocene
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_ziervogel_snow/" itemprop="url">Polysaccharide hydrolysis in the presence of oil and dispersants: Insights into potential degradation pathways of exopolymeric substances (EPS) from oil-degrading bacteria</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Oceanic oil-degrading bacteria produce copious amounts of exopolymeric substances (EPS) that facilitate their access to oil. The fate of EPS in the water column is in part determined by activities of heterotrophic microbes capable of utilizing EPS compounds as carbon and energy sources. To evaluate the potential of natural microbial communities to degrade EPS produced during oil degradation, we measured potential hydrolysis rates of six structurally distinct polysaccharides in two roller bottle experiments, using water from a natural oil seep in the northern Gulf of Mexico. The suite of polysaccharides used to measure the initial step in carbon degradation is indicative of polymers within microbial EPS. The treatments included (i) unamended surface or deep waters (whole water), and water amended with (ii) a water-accommodated fraction of oil (WAF), (iii) oil dispersant Corexit 9500, and (iv) WAF chemically-enhanced with Corexit (CEWAF). The oil and Corexit treatments were employed to simulate conditions during the Deepwater Horizon oil spill. Polysaccharide hydrolysis rates in the surface-water treatments were lowest in the WAF treatment, despite elevated levels of EPS in the form of transparent exopolymer particles (TEP). In contrast, the three deep-water treatments (WAF, Corexit, CEWAF) showed enhanced hydrolysis rates and TEP levels (WAF) compared to the whole water. We also observed variations in the spectrum of polysaccharide-hydrolyzing enzyme activities among the treatments. These substrate specificities were likely driven by activities of oil-degrading bacteria, shaping the pool of EPS and TEP as well as degradation products of hydrocarbons and Corexit compounds. A model calculation of potential turnover rates of organic carbon within the TEP pool suggests extended residence times of TEP in oil-contaminated waters, making them prone to serve as the sticky matrix for oily aggregates known as marine oil snow.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Ziervogel_snow.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_ziervogel_snow/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.elementascience.org/articles/10.1525/elementa.371/" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1525/elementa.371" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-08-06 00:06:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-09 14:09:10 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/troyalty/">Taylor Royalty</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>August 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      mSphere
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_royalty_steen_mags/" itemprop="url">Theoretical and Simulation-Based Investigation of the Relationship between Sequencing Effort, Microbial Community Richness, and Diversity in Binning Metagenome-Assembled Genomes</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    We applied theoretical and simulation-based approaches to characterize how microbial community structure influences the amount of sequencing effort to reconstruct metagenomes that are assembled from short-read sequences. First, a coupon collector equation was proposed as an analytical model for predicting sequencing effort as a function of microbial community structure. Characterization was performed by varying community structure properties such as richness, evenness, and genome size. Simulations demonstrated that while community richness and evenness influenced the sequencing effort required to sequence a community metagenome to exhaustion, the effort necessary to sequence an individual genome to a target fraction of exhaustion depended only on the relative abundance of the ge- nome and its genome size. A second analysis evaluated the quantity, completion, and contamination of complete-metagenome-assembled genome equivalents, a bioinformatic pipeline normalized metric for metagenome-assembled genome (MAG) quantity, as a function of sequencing effort on four preexisting sequence read data sets from different environments. These data sets were subsampled to various degrees of completeness to simulate the effect of sequencing effort on MAG retrieval. Modeling suggested that sequencing efforts beyond what is typical in published experiments (1 to 10 Gbp) would generate diminishing returns in terms of MAG bin- ning. A software tool, Genome Relative Abundance to Sequencing Effort (GRASE), was created to assist investigators to further explore this relationship. Reevaluation of the relationship between sequencing effort and binning success in the context of genome relative abundance, as opposed to base pairs, provides a constraint on sequencing experiments based on the relative abundance of microbes in an environ- ment rather than arbitrary levels of sequencing effort.
  </div>
  

  
  <div class="btn-links">
    








  





<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_royalty_steen_mags/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41396-019-0484-y" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-08-06 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-09 21:40:53 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>August 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      ISME Journal
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2019_steen_et_al_isme/">
      <img src="/publication/2019_steen_et_al_isme/featured_hufb273d8ecccadd1f77b053cfe1574efb_1751387_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_steen_et_al_isme/" itemprop="url">High proportions of bacteria and archaea across most biomes remain uncultured</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    A recent paper by Martiny argues that “high proportions” of bacteria in diverse Earth environments have been cultured. Here we reanalyze a portion of the data in that paper, and argue that the conclusion is based on several technical errors, most notably a calculation of sequence similarity that does not account for sequence gaps, and the reliance on 16S rRNA gene amplicons that are known to be biased towards cultured organisms. We further argue that the paper is also based on a conceptual error: namely, that sequence similarity cannot be used to infer “culturability” because one cannot infer physiology from 16S rRNA gene sequences. Combined with other recent, more reliable studies, the evidence supports the conclusion that most bacterial and archaeal taxa remain uncultured.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Steen_et_al_Uncultured.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_steen_et_al_isme/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41396-019-0484-y" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41396-019-0484-y" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-08-06 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-26 10:39:53 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/troyalty/">Taylor Royalty</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>August 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Applied and Environmental Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2019_steen_et_al_wor_enz/">
      <img src="/publication/2019_steen_et_al_wor_enz/featured_hu1656548ef733a7c2dffbc57534bef85b_1526248_918x517_fill_q90_lanczos_smart1.jpeg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_steen_et_al_wor_enz/" itemprop="url">Kinetics and Identities of Extracellular Peptidases in Subsurface Sediments of the White Oak River Estuary, North Carolina</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Anoxic subsurface sediments contain communities of heterotrophic microorganisms that metabolize organic carbon at extraordinarily slow rates. In order to assess the mechanisms by which subsurface microorganisms access detrital sedimentary organic matter, we measured kinetics of a range of extracellular peptidases in anoxic sediments of the White Oak River estuary, NC. Nine distinct peptidase substrates were enzymatically hydrolyzed at all depths. Potential peptidase activities (Vmax) decreased with increasing sediment depth, although Vmax expressed on a per cell basis was approximately the same at all depths. Half-saturation constants (Km) decreased with depth, indicating peptidases that functioned more efficiently at low substrate concentrations. Potential activities of extracellular peptidases acting on molecules that are enriched in degraded organic matter (D-phenylalanine and L-ornithine) increased relative to enzymes that act on L-phenylalanine, further suggesting microbial community adaptation to access degraded organic matter. Nineteen classes of predicted, exported peptidases were identified in genomic data from the same site, of which genes for class C25 (gingipain-like) peptidases represented more than 40% at each depth. Methionine aminopeptidases, zinc carboxypeptidases, and class S24-like peptidases, which are involved in single-stranded DNA repair, were also abundant. These results suggest a subsurface heterotrophic microbial community that primarily accesses low-quality detrital organic matter via a diverse suite of well-adapted extracellular enzymes.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Steen_et_al_WOR.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_steen_et_al_wor_enz/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/adsteen/WOR_enz_2013_2014" target="_blank" rel="noopener">
  Code
</a>












<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41396-019-0484-y" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-04-16 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-08 14:27:51 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>April 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2019_bird_et_al_baltic/">
      <img src="/publication/2019_bird_et_al_baltic/featured_hue972652b6cc45a27e2ff1d6c6b42f505_174914_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_bird_et_al_baltic/" itemprop="url">Uncultured microbial phyla suggest mechanisms for multi-thousand-year subsistence in Baltic Sea sediments</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Energy-starved microbes in deep marine sediments subsist at near-zero growth for thousands of years, yet the mechanisms for their subsistence are unknown because no model strains have been cultivated from most of these groups. We investigated Baltic Sea sediments with single-cell genomics, metabolomics, metatranscriptomics, and enzyme assays to identify possible subsistence mechanisms employed by uncultured Atribacteria, Aminicenantes, Actinobacteria group OPB41, Aerophobetes, Chloroflexi, Deltaproteobacteria, Desulfatiglans, Bathyarchaeota, and Euryarchaeota marine group II lineages. Some functions appeared to be shared by multiple lineages, such as trehalose production and NAD+-consuming deacetylation, both of which have been shown to increase cellular life spans in other organisms by stabilizing proteins and nucleic acids, respectively. Other possible subsistence mechanisms differed between lineages, possibly providing them different physiological niches. Enzyme assays and transcripts suggested that Atribacteria and Actinobacteria group OPB41 catabolized sugars, whereas Aminicenantes and Atribacteria catabolized peptides. Metabolite and transcript data suggested that Atribacteria utilized allantoin, possibly as an energetic substrate or chemical protectant, and also possessed energy-efficient sodium pumps. Atribacteria single-cell amplified genomes (SAGs) recruited transcripts for full pathways for the production of all 20 canonical amino acids, and the gene for amino acid exporter YddG was one of their most highly transcribed genes, suggesting that they may benefit from metabolic interdependence with other cells. Subsistence of uncultured phyla in deep subsurface sediments may occur through shared strategies of using chemical protectants for biomolecular stabilization, but also by differentiating into physiological niches and metabolic interdependencies.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Bird_et_al_Baltic.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_bird_et_al_baltic/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1128/mBio.02376-18" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2019-03-19 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-08 14:11:41 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/lquigley/">Lauren N. M. Quigley</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>March 2019</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2019_quigley_interactivity/" itemprop="url">Characterization of the Interactive Effects of Labile and Recalcitrant Organic Matter on Microbial Growth and Metabolism</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Geochemical models typically represent organic matter (OM) as consisting of multiple, independent pools of compounds, each accessed by microorganisms at different rates. However, recent findings indicate that organic compounds can interact within microbial metabolisms. The relevance of interactive effects within marine systems is debated and a mechanistic understanding of its complexities, including microbe-substrate relationships, is lacking. As a first step toward uncovering mediating processes, the interactive effects of distinct pools of OM on the growth and respiration of marine bacteria, individual strains and a simple, constructed community of Roseobacter lineage members were tested. Isolates were provided with natural organic matter (NOM) and different concentrations (1, 4, 40, 400 μM-C) and forms of labile OM (acetate, casamino acids, tryptone, coumarate). The microbial response to the mixed substrate regimes was assessed using viable counts and respiration in two separate experiments. Two marine bacteria and a six-member constructed community were assayed with these experiments. Both synergistic and antagonistic growth responses were evident for all strains, but all were transient. The specific substrate conditions promoting a response, and the direction of that response, varied amongst species. These findings indicate that the substrate conditions that result in OM interactive effects are both transient and species-specific and thus influenced by both the composition and metabolic potential of a microbial community.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2019_Quigley_interactive.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019_quigley_interactivity/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fmicb.2019.00493" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-04-13 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-05 11:25:09 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>April 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2018_matzek_persulfate/">
      <img src="/publication/2018_matzek_persulfate/featured_hu61f087974b8787240438d7e3bdb5403e_40954_918x517_fill_q90_lanczos_smart1.gif" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2018_matzek_persulfate/" itemprop="url">Understanding Electrochemically Activated Persulfate and Its Application to Ciprofloxacin Abatement</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    This study offers insight into the roles anodic and cathodic processes play in electrochemically activated persulfate (EAP) and screens EAP as a viable technique for ciprofloxacin degradation in wastewater. Sulfate radical formation at a boron-doped diamond (BDD) anode and persulfate activation at a graphite cathode were experimentally elucidated using different electrolytes and electrochemical setups. Rapid ciprofloxacin transformation occurred via pseudo-first-order mechanisms with respect to ciprofloxacin in persulfate electrolyte, reaching 84% removal in 120 min using EAP. Transformation pathways were compared to those in nitrate and sulfate electrolytes. Ciprofloxacin removal rates in the electrochemical system were 88% and 33% faster in persulfate than nitrate and sulfate electrolytes, respectively. Total organic carbon removal rates were 93% and 48% faster in persulfate than nitrate and sulfate, respectively. Use of sulfate electrolyte resulted in removal rates 6–7 times faster than those in nitrate solution. Accelerated removal in sulfate was attributed to anodic sulfate radical formation, while enhanced removal in persulfate was associated with cathodic persulfate activation and nonradical persulfate activation at the BDD anode. Quenching experiments indicated both sulfate radicals and hydroxyl radicals contributed to degradation. Comparisons between platinum and graphite cathodes showed similar cathodic persulfate activation and ciprofloxacin degradation.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2018_Matzek_persulfate_ciproflaxin.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2018_matzek_persulfate/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1021/acs.est.8b00015" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2018-03-06 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-06 09:40:34 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/lmullen/">Lauren Mullen</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>March 2018</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2018_mullen_et_al_mxshs/">
      <img src="/publication/2018_mullen_et_al_mxshs/featured_hu98982b0bb24fd2372dbd1414c5739cb7_1523384_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2018_mullen_et_al_mxshs/" itemprop="url">Potential Activities of Freshwater Exo- and Endo-Acting Extracellular Peptidases in East Tennessee and the Pocono Mountains</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Proteins constitute a particularly bioavailable subset of organic carbon and nitrogen in aquatic environments but must be hydrolyzed by extracellular enzymes prior to being metabolized by microorganisms. Activities of extracellular peptidases (protein-degrading enzymes) have frequently been assayed in freshwater systems, but such studies have been limited to substrates for a single enzyme [leucyl aminopeptidase (Leu-AP)] out of more than 300 biochemically recognized peptidases. Here, we report kinetic measurements of extracellular hydrolysis of five substrates in 28 freshwater bodies in the Delaware Water Gap National Recreation Area in the Pocono Mountains (PA, United States) and near Knoxville (TN, United States), between 2013 and 2016. The assays putatively test for four aminopeptidases (arginyl aminopeptidase, glyclyl aminopeptidase, Leu-AP, and pyroglutamyl aminopeptidase), which cleave N-terminal amino acids from proteins, and trypsin, an endopeptidase, which cleaves proteins mid-chain. Aminopeptidase and the trypsin-like activity were observed in all water bodies, indicating that a diverse set of peptidases is typical in freshwater. However, ratios of peptidase activities were variable among sites: aminopeptidases dominated at some sites and trypsin-like activity at others. At a given site, the ratios remained fairly consistent over time, indicating that they are driven by ecological factors. Studies in which only Leu-AP activity is measured may underestimate the total peptidolytic capacity of an environment, due to the variable contribution of endopeptidases.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2018_Mullen_et_al_PEEC_TN.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2018_mullen_et_al_mxshs/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fmicb.2018.00368" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2017-04-03 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-06 09:40:34 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>April 2017</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Environmental Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2017_mahmoudi_isocarb/">
      <img src="/publication/2017_mahmoudi_isocarb/featured_huc531a6adc0bdc9ea68a547e528248b80_74105_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2017_mahmoudi_isocarb/" itemprop="url">Sequential bioavailability of sedimentary organic matter to heterotrophic bacteria</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Aquatic sediments harbour diverse microbial communities that mediate organic matter degradation and influence biogeochemical cycles. The pool of bioavailable carbon continuously changes as a result of abiotic processes and microbial activity. It remains unclear how microbial communities respond to heterogeneous organic matrices and how this ultimately affects heterotrophic respiration. To explore the relationships between the degradation of mixed carbon substrates and microbial activity, we incubated batches of organic‐rich sediments in a novel bioreactor (IsoCaRB) that permitted continuous observations of CO2 production rates, as well as sequential sampling of isotopic signatures (δ13C, Δ14C), microbial community structure and diversity, and extracellular enzyme activity. Our results indicated that lower molecular weight (MW), labile, phytoplankton‐derived compounds were degraded first, followed by petroleum‐derived exogenous pollutants, and finally by higher MW polymeric plant material. This shift in utilization coincided with a community succession and increased extracellular enzyme activities. Thus, sequential utilization of different carbon pools induced changes at both the community and cellular level, shifting community composition, enzyme activity, respiration rates, and residual organic matter reactivity. Our results provide novel insight into the accessibility of sedimentary organic matter and demonstrate how bioavailability of natural organic substrates may affect the function and composition of heterotrophic bacterial populations.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2017_Mahmoudi_et_al-Environmental_Microbiology.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2017_mahmoudi_isocarb/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1111/1462-2920.13745" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2016-02-08 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-04 23:16:00 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>, <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/lquigley/">Lauren N. M. Quigley</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>February 2016</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Marine Science
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2016_steen_priming/">
      <img src="/publication/2016_steen_priming/featured_hu5e7906c749a30169fcebf0fddd6390e7_188702_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2016_steen_priming/" itemprop="url">Evidence for the Priming Effect in a Planktonic Estuarine Microbial Community</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The “priming effect,” in which addition of labile substances changes the remineralization rate of recalcitrant organic matter, has been intensively studied in soils, but is less well-documented in aquatic systems. We investigated the extent to which additions of nutrients or labile organic carbon could influence remineralization rates of 14C-labeled, microbially-degraded, phytoplankton-derived organic matter (OM) in microcosms inoculated with microbial communities drawn from Grove Creek Estuary in coastal Georgia, USA. We found that amendment with labile protein plus phosphorus increased remineralization rates of degraded, phytoplankton-derived OM by up to 100%, whereas acetate slightly decreased remineralization rates relative to an unamended control. Addition of ammonium and phosphate induced a smaller effect, whereas addition of ammonium alone had no effect. Counterintuitively, alkaline phosphatase activities increased in response to the addition of protein under P-replete conditions, indicating that production of enzymes unrelated to the labile priming compound may be a mechanism for the priming effect. The observed priming effect was transient: after 36 days of incubation roughly the same quantity of organic carbon had been mineralized in all treatments including no-addition controls. This timescale is on the order of the typical hydrologic residence times of well-flushed estuaries suggesting that priming in estuaries has the potential to influence whether OC is remineralized in situ or exported to the coastal ocean.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2016_Steen_Quigley_Buchan_priming.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2016_steen_priming/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fmars.2016.00006" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2015-06-20 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-04 14:07:57 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2015</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Aquatic Microbial Ecology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2015_steen_et_al_subs_specificity/" itemprop="url">Substrate specificity of aquatic extracellular peptidases assessed by competitive inhibition assays using synthetic substrates</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The identities and biochemical properties of extracellular enzymes present in natural environments are poorly constrained. We used a series of competitive inhibition experiments with samples from a freshwater environment (the Tennessee River at Knoxville, TN, USA) and a marine environment (Bogue Sound, NC, USA) to characterize the range of substrate specificities of naturally occurring enzymes that hydrolyze L-leucine 7-amido-4-methylcoumarin (Leu-AMC), L‑proline-AMC (Pro-AMC), and L-arginine-AMC (Arg-AMC)—putative substrates for leucyl-aminopeptidase, prolyl-aminopeptidase, and arginyl-aminopeptidase, respectively. Extracellular peptidases which hydrolyzed Arg-AMC and Leu-AMC demonstrated affinity for up to 8 other amino acids, whereas those hydrolyzing Pro-AMC in the Tennessee River, and Arg-AMC at Bogue Sound, were more specific to proline and arginine, respectively. Patterns of substrate affinity showed that Leu-AMC (at both sampling sites) and Arg-AMC (at Bogue Sound) were primarily hydrolyzed by enzymes other than leucyl-aminopeptidase and arginyl-aminopeptidase, respectively. The set of naturally occurring peptidases in both environments showed greater affinity towards a subset of amino acids. These amino acids were on average larger, yielded more free energy from oxidation to CO2, and tended to be depleted in aged organic matter. These relationships indicate that pathways of amino acid diagenesis are at least partially controlled by the substrate specificities of the peptidases involved in protein degradation.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2015_Steen_Vazin_et_al_2015_substrate_specificity.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2015_steen_et_al_subs_specificity/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3354/ame01755" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2015-06-10 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-04 17:29:16 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>June 2015</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      FASEB Journal
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2015_michalska_et_al/">
      <img src="/publication/2015_michalska_et_al/featured_huca046fa4896c492d48fe98b6e5dfaf4f_220553_918x517_fill_q90_lanczos_smart1.jpeg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2015_michalska_et_al/" itemprop="url">New aminopeptidase from “microbial dark matter” archaeon</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Marine sediments host a large population of diverse, heterotrophic, uncultured microorganisms with unknown physiologies that control carbon flow through organic matter decomposition. Recently, single-cell genomics uncovered new key players in these processes, such as the miscellaneous crenarchaeotal group. These widespread archaea encode putative intra- and extracellular proteases for the degradation of detrital proteins present in sediments. Here, we show that one of these enzymes is a self-compartmentalizing tetrameric aminopeptidase with a preference for cysteine and hydrophobic residues at the N terminus of the hydrolyzed peptide. The ability to perform detailed characterizations of enzymes from native subsurface microorganisms, without requiring that those organisms first be grown in pure culture, holds great promise for understanding key carbon transformations in the environment as well as identifying new enzymes for biomedical and biotechnological applications.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2015_Michalska_et_al_Bathyaminopeptidase.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2015_michalska_et_al/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1096/fj.15-272906" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2014-09-01 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-06 09:42:41 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>September 2014</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Biogeochemistry 117:5-21
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2013_arnosti_et_al_review/">
      <img src="/publication/2013_arnosti_et_al_review/featured_hu4b3918bb673c78649b7a84798085d766_80434_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2013_arnosti_et_al_review/" itemprop="url">Extracellular enzymes in terrestrial, freshwater, and marine environments: Perspectives on system variability and common research needs</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    Extracellular enzymes produced by heterotrophic microbial communities are major drivers of carbon and nutrient cycling in terrestrial, freshwater, and marine environments. Although carbon and nutrient cycles are coupled on global scales, studies of extracellular enzymes associated with terrestrial, freshwater, and marine microbial communities are not often compared across ecosystems. In part, this disconnect arises because the environmental parameters that control enzyme activities in terrestrial and freshwater systems, such as temperature, pH, and moisture content, have little explanatory power for patterns of enzyme activities in marine systems. Instead, factors such as the functional diversity of microbial communities may explain varying patterns of enzyme activities observed in the ocean to date. In any case, many studies across systems focus on similar issues that highlight the commonalities of microbial community organization. Examples include the effective lifetime of enzymes released into the environment; the extent to which microbial communities coordinate enzyme expression to decompose complex organic substrates; and the influence of microbial community composition on enzyme activities and kinetics. Here we review the often-disparate research foci in terrestrial, freshwater, and marine environments. We consider the extent to which environmental factors may regulate extracellular enzyme activities within each ecosystem, and highlight commonalities and current methodological challenges to identify research questions that may aid in integrating crosssystem perspectives in the future.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2014_Arnosti_review.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2013_arnosti_et_al_review/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/s10533-013-9906-5" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2014-04-11 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-04 17:29:16 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>April 2014</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2014_cardman_et_al_verrucomicrobia/" itemprop="url">Verrucomicrobia Are Candidates for Polysaccharide-Degrading Bacterioplankton in an Arctic Fjord of Svalbard</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    In Arctic marine bacterial communities, members of the phylum <em>Verrucomicrobia</em> are consistently detected, although not typically abundant, in 16S rRNA gene clone libraries and pyrotag surveys of the marine water column and in sediments. In an Arctic fjord (Smeerenburgfjord) of Svalbard, members of the Verrucomicrobia, together with Flavobacteria and smaller proportions of Alpha- and Gammaproteobacteria, constituted the most frequently detected bacterioplankton community members in 16S rRNA gene-based clone library analyses of the water column. Parallel measurements in the water column of the activities of six endo-acting polysaccharide hydrolases showed that chondroitin sulfate, laminarin, and xylan hydrolysis accounted for most of the activity. Several Verrucomicrobia water column phylotypes were affiliated with previously sequenced, glycoside hydrolase-rich genomes of individual Verrucomicrobia cells that bound fluorescently labeled laminarin and xylan and therefore constituted candidates for laminarin and xylan hydrolysis. In sediments, the bacterial community was dominated by different lineages of Verrucomicrobia, Bacteroidetes, and Proteobacteria but also included members of multiple phylum-level lineages not observed in the water column. This community hydrolyzed laminarin, xylan, chondroitin sulfate, and three additional polysaccharide substrates at high rates. Comparisons with data from the same fjord in the previous summer showed that the bacterial community in Smeerenburgfjord changed in composition, most conspicuously in the changing detection frequency of Verrucomicrobia in the water column. Nonetheless, in both years the community hydrolyzed the same polysaccharide substrates.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2014_Cardman_et_al_Verrucomicrobia.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2014_cardman_et_al_verrucomicrobia/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1128/AEM.00899-14" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="2013-10-04 00:00:00 &#43;0000 UTC" itemprop="datePublished">
<meta content="2019-09-04 14:07:57 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>October 2013</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Applied and Environmental Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2013_lloyd_et_al_metaanalysis/" itemprop="url">Meta-Analysis of Quantification Methods Shows that Archaea and Bacteria Have Similar Abundances in the Subseafloor</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The identities and biochemical properties of extracellular enzymes present in natural environments are poorly constrained. We used a series of competitive inhibition experiments with samples from a freshwater environment (the Tennessee River at Knoxville, TN, USA) and a marine environment (Bogue Sound, NC, USA) to characterize the range of substrate specificities of naturally occurring enzymes that hydrolyze L-leucine 7-amido-4-methylcoumarin (Leu-AMC), L‑proline-AMC (Pro-AMC), and L-arginine-AMC (Arg-AMC)—putative substrates for leucyl-aminopeptidase, prolyl-aminopeptidase, and arginyl-aminopeptidase, respectively. Extracellular peptidases which hydrolyzed Arg-AMC and Leu-AMC demonstrated affinity for up to 8 other amino acids, whereas those hydrolyzing Pro-AMC in the Tennessee River, and Arg-AMC at Bogue Sound, were more specific to proline and arginine, respectively. Patterns of substrate affinity showed that Leu-AMC (at both sampling sites) and Arg-AMC (at Bogue Sound) were primarily hydrolyzed by enzymes other than leucyl-aminopeptidase and arginyl-aminopeptidase, respectively. The set of naturally occurring peptidases in both environments showed greater affinity towards a subset of amino acids. These amino acids were on average larger, yielded more free energy from oxidation to CO2, and tended to be depleted in aged organic matter. These relationships indicate that pathways of amino acid diagenesis are at least partially controlled by the substrate specificities of the peptidases involved in protein degradation.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2013_Lloyd_et_al_quant_metaanalysis.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2013_lloyd_et_al_metaanalysis/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/doi:10.1128/AEM.02090-13" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="" itemprop="datePublished">
<meta content="2019-09-06 09:40:34 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 0001</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  
  
  <a href="/publication/2018_lloyd_et_al_uncultured/">
      <img src="/publication/2018_lloyd_et_al_uncultured/featured_hu483438cd82e3753239ac177b1c5d1640_114970_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" itemprop="image" alt="">
  </a>
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2018_lloyd_et_al_uncultured/" itemprop="url">Phylogenetically Novel Uncultured Microbial Cells Dominate Earth Microbiomes</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    To describe a microbe’s physiology, including its metabolism, environmental roles, and growth characteristics, it must be grown in a laboratory culture. Unfortunately, many phylogenetically novel groups have never been cultured, so their physiologies have only been inferred from genomics and environmental characteristics. Although the diversity, or number of different taxonomic groups, of uncultured clades has been studied well, their global abundances, or numbers of cells in any given environment, have not been assessed. We quantified the degree of similarity of 16S rRNA gene sequences from diverse environments in publicly available metagenome and metatranscriptome databases, which we show have far less of the culture bias present in primer-amplified 16S rRNA gene surveys, to those of their nearest cultured relatives. Whether normalized to scaffold read depths or not, the highest abundances of metagenomic 16S rRNA gene sequences belong to phylogenetically novel uncultured groups in seawater, freshwater, terrestrial subsurface, soil, hypersaline environments, marine sediment, hot springs, hydrothermal vents, nonhuman hosts, snow, and bioreactors (22% to 87% uncultured genera to classes and 0% to 64% uncultured phyla). The exceptions were human and human-associated environments, which were dominated by cultured genera (45% to 97%). We estimate that uncultured genera and phyla could comprise 7.3 × 10<!-- raw HTML omitted -->29<!-- raw HTML omitted --> (81%) and 2.2 × 10<!-- raw HTML omitted -->29<!-- raw HTML omitted --> (25%) of microbial cells, respectively. Uncultured phyla were overrepresented in metatranscriptomes relative to metagenomes (46% to 84% of sequences in a given environment), suggesting that they are viable. Therefore, uncultured microbes, often from deeply phylogenetically divergent groups, dominate nonhuman environments on Earth, and their undiscovered physiologies may matter for Earth systems.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2018_Lloyd_Steen_et_al_Uncultured_Majority.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2018_lloyd_et_al_uncultured/cite.bib">
  Cite
</button>













<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1128/mSystems.00055-18" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

      
    
      
        






  
  





  


<div class="card-simple" itemscope itemtype="http://schema.org/ScholarlyArticle">

  
    



<meta content="" itemprop="datePublished">
<meta content="2020-09-03 21:38:35 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  
  
  
  <div>
    



  <span itemprop="author name" itemtype="http://schema.org/Person"><a href="/authors/admin/">Drew Steen</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    <time>January 0001</time>
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Microbiology
    
  </span>
  

  

  
  
  

  
  

  

</div>

  

  
  
  

  <h3 class="article-title mb-1 mt-3" itemprop="name">
    <a href="/publication/2020_murray_roadmap/" itemprop="url">Roadmap for naming uncultivated Archaea and Bacteria</a>
  </h3>

  
  <div class="article-style" itemprop="articleBody">
    The assembly of single-amplified genomes (SAGs) and metagenome-assembled genomes (MAGs) has led to a surge in genome-based discoveries of members affiliated with Archaea and Bacteria, bringing with it a need to develop guidelines for nomenclature of uncultivated microorganisms. The International Code of Nomenclature of Prokaryotes (ICNP) only recognizes cultures as ‘type material’, thereby preventing the naming of uncultivated organisms. In this Consensus Statement, we propose two potential paths to solve this nomenclatural conundrum. One option is the adoption of previously proposed modifications to the ICNP to recognize DNA sequences as acceptable type material; the other option creates a nomenclatural code for uncultivated Archaea and Bacteria that could eventually be merged with the ICNP in the future. Regardless of the path taken, we believe that action is needed now within the scientific community to develop consistent rules for nomenclature of uncultivated taxa in order to provide clarity and stability, and to effectively communicate microbial diversity.
  </div>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/pdf/2020_Murray_roadmap.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020_murray_roadmap/cite.bib">
  Cite
</button>












  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41564-020-0733-x" target="_blank" rel="noopener">
  Source Document
</a>




  </div>
  

</div>

      
    

    

  </div>
</div>

    </div>
  </section>

  
  
  

  

  

  

  

  

  
  

  
  
  

  
  
  
  
  

  
  

  <section id="contact" class="home-section wg-contact   "  >
    <div class="container">
      




<div class="row contact-widget">
  <div class="col-12 col-lg-4 section-heading">
    <h1>Contact</h1>
    
  </div>
  <div class="col-12 col-lg-8">
    

    

    
    
      
      
    

    <div class="mb-3">
      <form name="contact" method="POST" action="https://formspree.io/asteen1@utk.edu">
        <div class="form-group form-inline">
          <label class="sr-only" for="inputName">Name</label>
          <input type="text" name="name" class="form-control w-100" id="inputName" placeholder="Name" required>
        </div>
        <div class="form-group form-inline">
          <label class="sr-only" for="inputEmail">Email</label>
          <input type="email" name="email" class="form-control w-100" id="inputEmail" placeholder="Email" required>
        </div>
        <div class="form-group">
          <label class="sr-only" for="inputMessage">Message</label>
          <textarea name="message" class="form-control" id="inputMessage" rows="5" placeholder="Message" required></textarea>
        </div>
        <button type="submit" class="btn btn-outline-primary px-3 py-2">Send</button>
      </form>
    </div>
    

    <ul class="fa-ul" itemscope>

      

      

      
      <li>
        <i class="fa-li fas fa-map-marker fa-2x" aria-hidden="true"></i>
        <span id="person-address" itemprop="address">717 Strong Hall, 1621 Cumberland Ave, Knoxville, TN 37917, USA</span>
      </li>
      

      

      
      <li>
        <i class="fa-li fas fa-calendar-check fa-2x" aria-hidden="true"></i>
        <a href="https://calendar.google.com/calendar/embed?mode=week&amp;src=andrew.decker.steen%40gmail.com&amp;ctz=America/New_York" target="_blank" rel="noopener">Drew Steen&#39;s schedule</a>
      </li>
      

      
      
      
      
      
        
      
      
      
      
      
        
      
      <li>
        <i class="fa-li fab fa-twitter fa-2x" aria-hidden="true"></i>
        <a href="https://twitter.com/Twitter" target="_blank" rel="noopener">DM Me</a>
      </li>
      

    </ul>

    
    <div class="d-none">
      <input id="map-provider" value="2">
      <input id="map-lat" value="35.957262">
      <input id="map-lng" value="-83.931100">
      <input id="map-dir" value="717 Strong Hall, 1621 Cumberland Ave, Knoxville, TN 37917, USA">
      <input id="map-zoom" value="16">
      <input id="map-api-key" value="">
    </div>
    <div id="map"></div>
    

  </div>
</div>

    </div>
  </section>



      

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js" integrity="sha256-aYTdUrn6Ow1DDgh5JTc3aDGnnju48y/1c8s1dgkYPQ8=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/languages/r.min.js"></script>
        
      

      
      
    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.js" integrity="sha512-lInM/apFSqyy1o6s89K4iQUKg6ppXEgsVxT35HbzUupEVRh2Eu9Wdl4tHj7dZO0s1uvplcYGmt3498TtHq+log==" crossorigin="anonymous"></script>
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script>
      const search_index_filename = "/index.json";
      const i18n = {
        'placeholder': "Search...",
        'results': "results found",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.16bbb3750feb7244c9bc409a5a4fe678.js"></script>

    






  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    © 2020 &middot; 

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" id="back_to_top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
